cmd/run.go

```diff
@@ -155,6 +155,8 @@ the data document with the following syntax:
 	runCommand.Flags().VarP(authorization, "authorization", "", "set authorization scheme")
 	runCommand.Flags().VarP(logLevel, "log-level", "l", "set log level")
 	runCommand.Flags().VarP(logFormat, "log-format", "", "set log format")
+	runCommand.Flags().StringArrayVar(&params.ConfigOverrides, "set", []string{}, "override config values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)")
```

Comment:
Nit: `use commas to specify multiple values` instead?

Reply:
I think the wording is because you can specify multiple of the flags (eg `--set foo=bar --set a.b.c=d`) and/or use commas to separate them (eg `--set foo=bar,a.b.c=d`).

That being said, it is kinda confusing. I wasn't sure the best way to describe that so I borrowed this wording from helm https://github.com/helm/helm/blob/master/cmd/helm/install.go#L209-L211

Reply:
I just noticed the argument descriptions were long compared to the others so I was trying to make them more consistent. Also, I was trying to make use active voice ("use x for y") instead of the more passive version ("can y with x").

Reply:
Got it. One sec.. will update them.